Dockerfile-python

```diff
@@ -1,5 +1,23 @@
-FROM python:3
-RUN pip install postgres
-RUN pip install psycopg2
+FROM python:3.6
+COPY requirements.txt /
+RUN pip install -r /requirements.txt
 RUN apt update
-RUN apt install -y postgresql 
\ No newline at end of file
+RUN apt install -y postgresql 
+
+RUN apt-get update \
+  && apt-get -y upgrade \
+  && apt-get -y install git build-essential libsqlite3-dev zlib1g-dev
```

Comment:
can we install `postgresql` here itself or do we need to install separately?

Reply:
we need to install it separately.